lib/chitragupta/util.js

```diff
@@ -0,0 +1,76 @@
+const CLS = require('./cls');
+const CATEGORIES = require('./categories');
+const FORMAT_VERSIONS = require('./format_versions');
+
+function populateServerData(dataParam) {
+  const data = dataParam;
+  const request = CLS.get('request');
+  const response = CLS.get('response');
+  data.data.request = {};
+  data.data.response = {};
+
+  const { url } = request;
+  const [endpoint, params] = url.split('?');
```

Comment:
can there be multiple ? in the URL, for example `https://www.google.com/search?q=?` is a valid url

Reply:
For url like `https://www.google.com/search?q=?213` it will only consider `q=` in the params. Changing this logic to split only on the first `?`

Reply:
yes